crates/ironrdp-session/src/fast_path.rs

```diff
@@ -36,6 +37,7 @@ pub struct Processor {
     use_system_pointer: bool,
     mouse_pos_update: Option<(u16, u16)>,
     no_server_pointer: bool,
+    pointer_software_rendering: bool,
```

Comment:
Note: I would be interested in some doc comment explaining what is `use_system_pointer` as opposed to `pointer_software_rendering`.

Reply:
I'll add doc comment in `ProcessorBuilder`